modules/haloIdSystem.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * This module adds Audigent HaloId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/haloIdSystem
+ * @requires module:modules/userId
+ */
+
+import {submodule} from '../src/hook.js';
+
+/** @type {Submodule} */
+export const haloIdSubmodule = {
+  /**
+   * Used to link submodule with config
+   * @type {string}
+   */
+  name: 'haloId',
+  /**
+   * Decode the stored id value for passing to bid requests
+   * @function
+   * @param {string} value
+   * @returns {{haloid:string}}
+   */
+  getHaloId(callback) {
+    var script = document.createElement('script')
+    script.type = 'text/javascript';
+
+    script.onload = function() {
+      callback(window.localStorage.getItem('auHaloId'));
```

Comment:
In example storage type is mentioned as cookie but no cookie is set in the code.
Looks like HaloID is being set in local-storage even when storage type is cookie.
@idettman can pub control the local-storage usage through PBJS?

Reply:
Should use the methods in src/storageManager.js for local storage actions.